ferrilata-devilline.IntegrationTests/Scenarios/ApiGetPitchesTests.cs

```diff
@@ -0,0 +1,84 @@
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using ferrilata_devilline.IntegrationTests.Fixtures;
+using ferrilata_devilline.Models;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace ferrilata_devilline.IntegrationTests.Scenarios
+{
+    [Collection("BaseCollection")]
+    public class IntegrationTests
+    {
+        private readonly TestContext _testContext;
+
+        public IntegrationTests(TestContext testContext)
+        {
+            _testContext = testContext;
+        }
+
+        [Theory]
+        [InlineData("/api/pitches")]
```

Comment:
theories with only one InlineData can be simplified to Facts, I think we should?

Reply:
Right, fixed.